Week1/assignments/Ruphine-Kengne/palindrome.js

```diff
@@ -0,0 +1,19 @@
+//Check whether a string is a palindrome
+//Compare the string to lowercase
+//We will split the string into characters, reverse the array of characters, join the characters back to form a string,
+//and then test the created string against what was originally received
+
+const isPalindrome = str => {
+    if (str.toLowerCase() === str.toLowerCase().split('').reverse().join('')) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
You could just do a 1 liner:
`return str.toLowerCase() === str.toLowerCase().split('').reverse().join('')`

Reply:
Okay, thanks